android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -188,11 +189,11 @@ class QuestionnaireActivity : BaseMultiLanguageActivity() {
           )
         }
 
-      if (launchContextResources.isNotEmpty()) {
-        questionnaireFragmentBuilder.setQuestionnaireLaunchContexts(
-          launchContextResources.map { it.json() },
-        )
-      }
+//      if (launchContextResources.isNotEmpty()) {
+//        questionnaireFragmentBuilder.setQuestionnaireLaunchContexts(
+//          launchContextResources.map { it.json() },
+//        )
+//      }
```

Comment:
can you explain this please? thank you

Reply:
resolved, its wasn't map as per main updates required